app/models/product.rb

```diff
@@ -9,7 +9,9 @@ class Product < ApplicationRecord
   validates :name, presence: true, length: { minimum: 5 }
   validates :price, presence: true
   validates :link, presence: true
-  validates :novelty, numericality: { greater_than: 0, less_than_or_equal_to: 5 }
+  validates :novelty,
+            numericality: { greater_than: 0, less_than_or_equal_to: 5 },
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.